.github/CODEOWNERS

```
@@ -1,5 +1,5 @@
 # Default
-*                        @jhendrixMSFT
+*                        @chlowell @jhendrixMSFT @RickWinter
 
 # Git Hub bot rules
 /.github/fabricbot.json  @AlexGhiondea @jsquire
```

While we're at it? 😞 
```suggestion
/.github/fabricbot.json  @jsquire
```

---

Fixed